src/pages/Admin/Venue/VenueAdminPage.tsx

```diff
@@ -77,24 +66,7 @@ export const VenueAdminPage: React.FC = () => {
           <AnnouncementMessage />
         </div>
       </div>
-      <div className="VenueAdminPage__settings">
-        {isBannerAdminVisibile && (
-          <BannerAdmin
-            venueId={spaceId}
-            venue={space}
-            onClose={hideBannerAdmin}
-          />
-        )}
-
-        {!isBannerAdminVisibile && (
-          <>
-            <AnnouncementOptions
-              banner={space.banner}
-              onEdit={showBannerAdmin}
-            />
-          </>
-        )}
-      </div>
+      <div className="VenueAdminPage__settings"></div>
```

Comment:
Is this `className` still used for style? If so, I can suggest to simplify the name to something like `className={TW.settings}`, but considering this should be an admin page full of Tailwind, it might just be good enough to put: 
```suggestion
      <div data-bem="VenueAdminPage__settings"></div>
```
and move this page to `src/components/admin`

Reply:
Removed completely in https://github.com/sparkletown/sparkle/pull/2952/commits/38d6ac6a94dd7aa2549bf7fcb0ad9d2fc7a05a13